engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/errata.controller.js

```diff
@@ -82,7 +82,7 @@ angular.module('Bastion.errata').controller('ErrataController',
         });
 
         $scope.goToNextStep = function () {
-            IncrementalUpdate.setBulkErrata(nutupane.getAllSelectedResults());
+            IncrementalUpdate.setBulkErrata(nutupane.getAllSelectedResults('errata_id'));
```

Comment:
After our discussion the other day, I merged these into this PR. Do you want to keep them separate. In which case I will revert the common changes.